.travis.yml

```diff
@@ -1,13 +1,41 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
+  - if [[ $TRAVIS_PULL_REQUEST != false ]]; then
+      git pull -q --no-edit origin develop;
+    fi;
+  - if [[ $PYDATA == true ]]; then
+      git pull -q --no-edit origin deploy-pydata;
```

Comment:
I have removed the fix for #1066. Also, I am moving the discussion out of this comment thread.